src/client/Client.cc

```diff
@@ -1973,6 +1973,10 @@ int Client::make_request(MetaRequest *request,
       session = mds_sessions.at(mds);
     }
 
+    if (!session->mds_features.test(CEPHFS_FEATURE_OP_GETVXATTR)) {
+      return -CEPHFS_EOPNOTSUPP;
+    }
```

Comment:
Ah, this needs moved to `Client::_getvxattr`, no?

Reply:
No, because only when it's in the `Client::make_request()` will it choose a proper MDS. Before that we don't know which session should we check the feature bit.